code/modules/mob/observer/ghost/ghost.dm

```diff
@@ -421,8 +421,9 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	show_browser(src, dat, "window=manifest;size=370x420;can_close=1")
 
 //This is called when a ghost is drag clicked to something.
-/mob/observer/ghost/MouseDrop(atom/over)
+/mob/observer/ghost/MouseDrop(over_object, src_location, over_location, src_control, over_control, params)
 	SHOULD_CALL_PARENT(FALSE)
+	var/atom/over = over_object
```

Comment:
I don't understand why you can't just have the first argument here be `atom/over_object`?

Reply:
I mean, it's fine to have types in proc arguments that differ from their corresponding parent proc argument. I just wasn't sure why it was done this way, but you explained.